src/GitHub.InlineReviews/Tags/InlineCommentTagger.cs

```diff
@@ -50,7 +50,14 @@ public sealed class InlineCommentTagger : ITagger<InlineCommentTag>, IDisposable
             this.sessionManager = sessionManager;
         }
 
-        public bool ShowMargin => file != null;
+        public bool ShowMargin
+        {
+            get
+            {
+                var diff = file?.Diff;
```

Comment:
Nit: this could be rewritten more simply: `file?.Diff?.Count > 0`

Reply:
Thanks. I learn something every day! 👍